### ISSUE TITLE
Support for using the LSP

### ISSUE THREAD

**Issue by BrendanThompson** [Eiara/HCL.novaextension#11]
It would be good if this plugin was able to use https://github.com/hashicorp/terraform-ls as it will help with autocompletion and the likes. If I were better at JS I would help ☹️

**Comment by aurynn** [comment]
Yep, supporting the language server is something I'd like to do, just haven't had the chance to dig into it yet 🙂

**Comment by Vermyndax** [comment]
I'd also be willing to help if I had a clue about what I'm doing here.
